### PR TITLE
CBBF-109: FHIR Mapping Change (HHA)

### PR DESCRIPTION
**Pull request by jcgeer**
- Mapping REV_UNIT for Outpatient, Hospice, and HHA
- Pointing System and URL for DME_UNIT/MTUS_IND in CarrierClaimn back to
MTUS_CNT/UNIT_IND (undoing change for Carrier in CBBF-110).

### CODE REVIEW

File: bluebutton-server-app/src/main/java/gov/hhs/cms/bluebutton/server/app/stu3/providers/HHAClaimTransformer.java [hunk 1]
```diff
@@ -159,14 +159,6 @@ private static ExplanationOfBenefit transformClaim(HHAClaim claimGroup) {
 								claimLine.getRevCntr1stAnsiCd().get()));
 			}
 
-			item.addAdjudication()
```
Comment (glovercloud) [inline bluebutton-server-app/src/main/java/gov/hhs/cms/bluebutton/server/app/stu3/providers/HHAClaimTransformer.java]: No big deal but you might want to run this code through the Eclipse formatter, it looks a little off to me.

File: bluebutton-server-app/src/main/java/gov/hhs/cms/bluebutton/server/app/stu3/providers/HospiceClaimTransformer.java [hunk 2]
```diff
@@ -179,20 +179,15 @@ private static ExplanationOfBenefit transformClaim(HospiceClaim claimGroup) {
 					.setCode(TransformerConstants.CODED_MONEY_USD)
 					.setValue(claimLine.getBenficiaryPaymentAmount());
 
-			item.addAdjudication()
```
Comment (glovercloud) [inline bluebutton-server-app/src/main/java/gov/hhs/cms/bluebutton/server/app/stu3/providers/HospiceClaimTransformer.java]: Formatting again??

File: bluebutton-server-app/src/main/java/gov/hhs/cms/bluebutton/server/app/stu3/providers/HHAClaimTransformer.java (hunk 1 above)
Comment (jcgeer) [reply bluebutton-server-app/src/main/java/gov/hhs/cms/bluebutton/server/app/stu3/providers/HHAClaimTransformer.java]: Which line are you meaning, specifically? I haven't updated 162-169, just removing them from the code.

Comment (glovercloud) [reply bluebutton-server-app/src/main/java/gov/hhs/cms/bluebutton/server/app/stu3/providers/HHAClaimTransformer.java]: Haha, man I need coffee.  Please ignore.

File: bluebutton-server-app/src/main/java/gov/hhs/cms/bluebutton/server/app/stu3/providers/HospiceClaimTransformer.java (hunk 2 above)
Comment (glovercloud) [reply bluebutton-server-app/src/main/java/gov/hhs/cms/bluebutton/server/app/stu3/providers/HospiceClaimTransformer.java]: Same as above.

**Pull request merged**